canvas_manage_course/settings/base.py

```diff
@@ -103,8 +93,7 @@
     'async': 'default',
     'auth': 'default',
     'contenttypes': 'default',
-    'icommons_common': 'termtool',
-    'lti_permissions': 'default',  # deprecated, but still around for migrations
+    'icommons_common': 'coursemanager',
```

Comment:
What do you think about converting over to the `CourseSchema` database router instead of this one, given that w'er only mapping `icommons_common` models to the `coursemanager` db?